source/extensions/access_loggers/filters/cel/cel.h

```diff
@@ -21,15 +21,14 @@ class CELAccessLogExtensionFilter : public AccessLog::Filter {
 public:
   CELAccessLogExtensionFilter(const ::Envoy::LocalInfo::LocalInfo& local_info,
                               Extensions::Filters::Common::Expr::BuilderInstanceSharedPtr,
-                              const google::api::expr::v1alpha1::Expr&);
+                              const cel::expr::Expr&);
 
   bool evaluate(const Formatter::HttpFormatterContext& log_context,
                 const StreamInfo::StreamInfo& stream_info) const override;
 
 private:
   const ::Envoy::LocalInfo::LocalInfo& local_info_;
   Extensions::Filters::Common::Expr::BuilderInstanceSharedPtr builder_;
-  const google::api::expr::v1alpha1::Expr parsed_expr_;
```

Comment:
We have to own parsed expression because the compiler puts view on the original proto. This would cause invalid memory access without a stored proto.

Reply:
Good catch! Let me fix it